test/validation/api/ipsec/ipsec.c

```diff
@@ -525,29 +523,41 @@ static int send_pkts(const ipsec_test_part part[], int num_part)
 	return num_part;
 }
 
+static odp_event_t plain_queue_deq(odp_queue_t queue, uint64_t wait_ns)
+{
+	odp_time_t cur, wait, next;
+	odp_event_t event;
+
+	if (wait_ns == 0)
+		return odp_queue_deq(queue);
```

Comment:
This if statement seems unnecessary to me as the timeout loop could handle zero timeouts too (if you change the > to >=).

Reply:
Ack.